analyze.py

```diff
@@ -43,15 +45,19 @@ def run(args_dict):
         args_dict['date'].append(args_dict['date'][0])
     args_dict['date'] = [datetime.strptime(x, '%m/%d/%Y') for x in args_dict['date']]
 
-    db = mdb.connect(user='{}'.format(USERNAME), host=args_dict['host'],
-                     password='{}'.format(PASSWORD), db=args_dict['db'], charset='utf8')
+    if args_dict['dbengine'] == 'mysql':
+        db = mdb.connect(user='{}'.format(USERNAME), host=args_dict['host'],
+                         password='{}'.format(PASSWORD), db=args_dict['db'], charset='utf8')
+    elif args_dict['dbengine'] == 'sqlite':
+        db = sqlite3.connect('{}.db'.format(args_dict['db']), isolation_level=None)
 
     df = pdsql.read_sql('select * from {table}'.format(table=args_dict['table']), db)
 
     df.date = df.date.apply(lambda x: datetime.strptime(x, '%m/%d/%Y'))
 
     df = df[(df.date>=args_dict['date'][0]) & (df.date<=args_dict['date'][1])]
-
+    df.start.apply(lambda x: x/np.timedelta64(1, 'm'))
+    df.end.apply(lambda x: x/np.timedelta64(1, 'm'))
```

Comment:
These two apply statements -- please delete. These will be addressed shortly in a PR.